examples/printerdemo_mcu/README.md

```diff
@@ -1,2 +1,56 @@
 This is a "fork" of the [printer demo](../printerdemo/) modified to run on a smaller screen
-See the [MCU backend Readme](../../internal/backends/mcu/) for more info
+
+# Microcontroller Setup
+
+See [MCU setup](../../docs/mcu_setup.md) guide for extra setup steps needed for
+microcontroller devices.
+
+# Run the demo:
+
+## The simulator
+
+```sh
+cargo run -p printerdemo_mcu --features=mcu-simulator --release
+```
+
+## On the Raspberry Pi Pico
+
+## Using probe-run
+
+This require [probe-run](https://github.com/knurling-rs/probe-run) (`cargo install probe-run`)
```

Comment:
It also requires flip-link otherwise you end up with:
`
error: linker `flip-link` not found
  |
  = note: No such file or directory (os error 2)

error: could not compile `printerdemo_mcu` due to previous error
`
so: `cargo install flip-link`

Reply:
Good catch, thank you! I have that installed locally and missed this.